packages/components/tests/dummy/app/router.js

```diff
@@ -49,4 +50,7 @@ Router.map(function () {
     this.route('disclosure');
     this.route('interactive');
   });
+  this.route('helpers', function () {
+    this.route('gutters');
+  });
```

Comment:
Let's discuss about the pages organization, there are reasons why we may want to consider an alternative content structure.